docs/tutorials/timeseries/forecasting-indepth.md

```diff
@@ -0,0 +1,248 @@
+determines the length of the forecast horizon.
+
+![ ](https://autogluon-timeseries-datasets.s3.us-west-2.amazonaws.com/public/figures/forecasting-indepth1.png)
+:width:`600px`
+
+
+The [predict](https://auto.gluon.ai/stable/api/autogluon.predictor.html#autogluon.timeseries.TimeSeriesPredictor.predict) method of a `TimeSeriesPredictor` generates two types of forecasts:
+
+- **mean forecast** represents the expected value of the time series at each time step in the forecast horizon.
+- **quantile forecast** represents the quantiles of the forecast distribution.
+For example, if the `0.1` quantile (also known as P10) is equal to `x`, it means that the time series value is predicted to be below `x` 10% of the time. As another example, the `0.5` quantile (P50) corresponds to the median forecast.
+
+The quantiles can be used to reason about the range of possible outcomes.
+For instance, by the definition of the quantiles, the time series is predicted to be between the P10 and P90 values with 80% probability.
+
+
+![ ](https://autogluon-timeseries-datasets.s3.us-west-2.amazonaws.com/public/figures/forecasting-indepth2.png)
+:width:`600px`
+
+By default, the predictor outputs the quantiles `[0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]`. You can train the predictor with custom quantiles using the `quantile_levels` argument
+```python
+predictor = TimeSeriesPredictor(quantile_levels=[0.05, 0.5, 0.95])
+```
+
+## What forecasting models are available in AutoGluon?
+Forecasting models in AutoGluon can be divided into three broad categories: local, global, and ensemble models.
+
+**Local models** are simple statistical models that are specifically designed to capture patterns such as trend or seasonality.
+Despite their simplicity, these models often produce reasonable forecasts and serve as a strong baseline.
+Available local models include:
+
+- `ETS`
+- `ARIMA`
+
+If the dataset consists of multiple time series, we fit a separate local model to each time series — hence the name "local".
+This means, if we want to make a forecast for a new time series that wasn't part of the training set, all local models will be fit from scratch for the new time series.
+
+**Global models** are deep-learning-based algorithms that learn a single model from the entire training set consisting of multiple time series.
+AutoGluon relies on [GluonTS](https://ts.gluon.ai/stable/) for the implementation of global models in PyTorch and MXNet.
+Available global models include:
+
+- `DeepAR`
+- `SimpleFeedForward`
+- `Transformer`
+- `MQRNN`
+- `MQCNN`
+- `TemporalFusionTransformer`
+
+Finally, an **ensemble** model works by combining predictions of all other models.
+By default, `TimeSeriesPredictor` always fits a `WeightedEnsemble` on top of other models.
+This can be disabled by setting `enable_ensemble=False` when creating the predictor.
+
+For a list of tunable hyperparameters for each model, their default values, and other details see [Model zoo](#TODO).
```

Comment:
TODO?

Reply:
The plan is to create a separate page in the documentation, where we list all available models and their hyperparameters. This link here (#TODO currently) will point to that document.